mc2/controllers/docker/tests/test_docker_controller.py

```diff
@@ -83,7 +83,8 @@ def check_and_clear_appdata(appdata, controller):
     assert appdata.pop("cpus") == controller.marathon_cpus
     assert appdata.pop("mem") == controller.marathon_mem
     assert appdata.pop("instances") == controller.marathon_instances
-    check_and_remove_optional(appdata, "cmd", controller.marathon_cmd)
+    if controller.marathon_cmd:
+        check_and_remove_optional(appdata, "args", [controller.marathon_cmd])
```

Comment:
Checking for the truth-ness of `controller.marathon_cmd` and then passing to `check_and_remove_optional` is a bit confusing...